answerking-api/.aws-sam/build.toml

```diff
@@ -0,0 +1,12 @@
+# This file is auto generated by SAM CLI build command
```

Comment:
gitignore the .aws-sam folder.
https://www.toptal.com/developers/gitignore/api/sam